src/Distech.CloudRelay.Functions/Distech.CloudRelay.Functions.csproj

```diff
@@ -1,11 +1,11 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Not sure what the change actually is here

Reply:
I don't know. Beyond Compare does not report any changes 🤔